lua/cfc_entity_stubber/main_loader.lua

```diff
@@ -0,0 +1,79 @@
+cfcEntityStubber = { }
+cfcEntityStubber.stubQueue = { }
+cfcEntityStubber.stubbers = { }
+cfcEntityStubber.oldWeaponStats = { }
+cfcEntityStubber.newWeaponStats = { }
+local stubbersDirectory = "cfc_entity_stubber/stubbers/"
+
+-- MAIN STUBBER FUNCTIONS 
+function cfcEntityStubber.includeStubbers( )
+    local files = file.Find( stubbersDirectory .. "*.lua", "LUA", "datedesc" )
+
+    for _, stubber in ipairs( files ) do
+        MsgC( Color( 41, 41, 41 ), "[", Color( 150, 150, 150 ), "Stubber", Color( 41, 41, 41 ), "] ", Color( 0, 255, 0 ), stubber .. " successfully loaded.\n" )
```

Comment:
might be usefull to extract the these out into a function that applies the colors and prefixes for you.
at the very least make the colors variables

Reply:
Will do